src/redux/reducer.js

```diff
@@ -42,6 +43,7 @@ const initialState = {
     computedLanguage: getLocalStorageComputedLanguage(),
     currentChildren: null,
     selectedDirectory: null,
+    selectedDirectoryByContextualMenu: null,
```

Comment:
Not fan of this naming.  I would say the 'ByContextualMenu' is too much precise over a specific user action.
If tomorrow we delete a directory with the keyboard for example, there will be 'selectedDirectoryByContextualMenu=null'
I would prefer `activeDirectory` which is not equivalent as `selectedDirectory`, but actually the former `selectedDirectory` should become `activeDirectory` and the new one `selectedDirectory` but it could  be ambiguous...

Reply:
ok good idea